wgsl/index.bs

```diff
@@ -12805,7 +12805,7 @@ but a value may infer the type.
 
     Returns the `__modf_result_abstract` built-in structure, defined as follows:
     ```rust
-struct __modf_result {
+struct __modf_result_abstract {
```

Comment:
I think `_f32` suffix is better but could it make sense to omit `_abstract`?

Reply:
@dneto0 I kind of like how `__atomic_compare_exchange_result<T>` does it (where we could contraint T to be AbstractFloat), but it is fine either way, esp. if it is thought that no `
_abstract` adds ambiguity despite members.

Reply:
(oops didn't notice this got merged)